src/containers/ProfileTwoFactorAuth/index.tsx

```diff
@@ -48,13 +48,15 @@ class ProfileTwoFactorAuthComponent extends React.Component<Props, ProfileTwoFac
                                   : <FormattedMessage id="page.body.profile.header.account.content.twoFactorAuthentication.message.disable" />}
                     </span>
                 </label>
-                <Checkbox
-                    checked={is2faEnabled}
-                    className={'pg-profile-page__switch'}
-                    onChange={() => this.handleToggle2fa()}
-                    label={''}
-                    slider={true}
-                />
+                <Form>
+                    <Form.Check
+                        type="switch"
+                        id="2fa-switch"
+                        label=""
+                        onChange={() => this.handleToggle2fa()}
```

Comment:
no need to put arrow function in this onChange property, you can just put it as `onChange={this.handleToggle2fa}`
this arrow functions slow down the app performance